setup.py

```diff
@@ -44,9 +44,6 @@
 setup(
     name="hummingbird-ml",
     version=version_str,
-    description="Convert trained traditional machine learning models into tensor computations",
-    long_description=long_description,
```

Comment:
Hello thanks for your PR!  The goal is not to delete this text entirely, but to make it legal with the new-style setup.py files.  Something like this:

![Screenshot 2024-03-11 at 10 01 18 AM](https://github.com/microsoft/hummingbird/assets/1703543/754a26f6-a421-4dcc-9ad2-069b044cdf47)

Reply:
Please see my comment in the issue "For now just deleted these 3 lines, but we should change the format away from markdown. See [docs](https://pypi.org/help/#description-content-type)" ...